code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -126,10 +116,24 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	if(!summoner)
 		to_chat(src, "<span class='holoparasite bold'>For some reason, somehow, you have no summoner. Please report this bug immediately.</span>")
 		return
-	to_chat(src, "<span class='holoparasite'>You are <font color=\"[namedatum.colour]\"><b>[real_name]</b></font>, bound to serve [summoner.real_name].</span>")
+	to_chat(src, "<span class='holoparasite'>You are a <b<[real_name]</b>, bound to serve [summoner.real_name].</span>")
```

Comment:
you are a John

I don't think 'a' should have been added here unless there's something I'm missing

Reply:
this only appears when first spawned, before you can choose your name